go.mod

```diff
@@ -1,13 +1,15 @@
 module github.com/hashicorp/packer-plugin-vsphere
 
-go 1.19
+go 1.21
```

Comment:
🤔 `.go-version` is set at 1.21.8.

Should this be:

go 1.21

toolchain go1.12.8

❔

Reply:
Good catch, yeah, should be 1.21.8